be2-scala/src/test/scala/ch/epfl/pop/tests/MessageCreationUtils.scala

```diff
@@ -0,0 +1,47 @@
+package ch.epfl.pop.tests
+
+
+import ch.epfl.pop.crypto.Hash
+import ch.epfl.pop.json._
+import spray.json.enrichAny
+import com.google.crypto.tink.subtle.Ed25519Sign.KeyPair
+import com.google.crypto.tink.subtle.Ed25519Sign
+import ch.epfl.pop.json.JsonCommunicationProtocol.MessageContentDataFormat
+import java.nio.charset.StandardCharsets.UTF_8
+
+import java.util.Base64
+
+object MessageCreationUtils {
+
+  def b64Encode(b: Array[Byte]): Array[Byte] = Base64.getEncoder.encode(b)
+
+  def b64EncodeToString(b: Array[Byte]): String = Base64.getEncoder.encodeToString(b)
```

Comment:
Shouldn't this return a Base64String?

Reply:
Yes but I didn't touch the tests yet since everything will change anyways